### PR TITLE
RHDEVDOCS-2524 Logging Release Notes, Known Issues, and Bug Fixes

### PR DESCRIPTION
**Pull request by rolfedh**
Move logging-related release notes from OpenShift RNs to the new OpenShift Logging product, similar to what was done for Service Mesh.
[enterprise-4.7] only

### CODE REVIEW

File: _topic_map.yml [hunk 1]
```diff
@@ -1633,35 +1633,37 @@ Name: Logging
 Dir: logging
 Distros: openshift-enterprise,openshift-origin,openshift-dedicated
 Topics:
-- Name: About cluster logging
+- Name: Release notes
+  File: cluster-logging-release-notes
+- Name: About Logging
```
Comment (mburke5678) [inline _topic_map.yml:1638]: Shouldn't these topic map references be {ProtuctTitle} Logging? This distinguishes the cluster logging feature from the pod logging that comes default with OCP.

File: _topic_map.yml [hunk 2]
```diff
@@ -1633,35 +1633,37 @@ Name: Logging
...
+- Name: Installing the Logging and Elasticsearch Operators
   File: dedicated-cluster-deploying
   Distros: openshift-dedicated
-- Name: Configuring your cluster logging deployment
+- Name: Configuring your Logging deployment
   Dir: config
   Distros: openshift-enterprise,openshift-origin
   Topics:
-  - Name: About the Cluster Logging Custom Resource
+  - Name: About the Logging Custom Resource
```
Comment (mburke5678) [inline _topic_map.yml:1650]: s/Custom Resource/custom resource (my error not to have updated this)

File: release_notes/ocp-4-7-release-notes.adoc [hunk 3]
```diff
@@ -2049,15 +1957,7 @@ If a new node is being added to a Machine Config Pool that includes SR-IOV, this
...
-* Fluentd pods with the `ruby-kafka-1.1.0` and `fluent-plugin-kafka-0.13.1` gems are not compatible with Apache Kafka version 0.10.1.0.
-+
-As a result, log forwarding to Kafka fails with a message: `error_class=Kafka::DeliveryFailed error="Failed to send messages to flux-openshift-v4/1"`
-+
-The `ruby-kafka-0.7` gem dropped support for Kafka 0.10 in favor of native support for Kafka 0.11. The `ruby-kafka-1.0.0` gem added support for Kafka 2.3 and 2.4. The current version of OpenShift Logging tests and therefore supports Kafka version 2.4.1.
-+
-To work around this issue, upgrade to a supported version of Apache Kafka.
-+
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1907370[*BZ#1907370*])
+* Fluentd pods with the `ruby-kafka-1.1.0` and `fluent-plugin-kafka-0.13.1` gems are not compatible with Apache Kafka version 0.10.1.0. For more information, see xref:../logging/cluster-logging-release-notes.adoc[Red Hat OpenShift Logging 5.0 release notes]
```
Comment (mburke5678) [inline release_notes/ocp-4-7-release-notes.adoc:1960]: Do we want to link this to Known Issues? https://deploy-preview-29785--osdocs.netlify.app/openshift-enterprise/latest/logging/cluster-logging-release-notes.html#openshift-logging-5-0-known-issues

**Comment by mburke5678** [comment]
@rolfedh A couple of questions. Let me know when it is ready for merge.

File: _topic_map.yml (hunk 1 above)
Comment (rolfedh) [reply _topic_map.yml:1638]: Discussed w. Christian, @sichvoge, and agreed on "Logging"

File: _topic_map.yml (hunk 2 above)
Comment (rolfedh) [reply _topic_map.yml:1650]: Updated.

File: release_notes/ocp-4-7-release-notes.adoc (hunk 3 above)
Comment (rolfedh) [reply release_notes/ocp-4-7-release-notes.adoc:1960]: It seems like the RNs prefer using `xref:` to link to docs and `link:` to link to bz's.

**Pull request merged**